src/WebApp/flask/app.py

```diff
@@ -249,19 +288,24 @@ def operationalization_post_operation(operation):
     	}
 
         mm_response = model_management.post('models', payload)
-        resp = Response(mm_response.text)
+        resp = Response(mm_response.text, status = mm_response.status_code)
         resp.headers['Content-type'] = 'application/json'
         return resp
     elif operation == 'registermanifest':
-        model_id = request.form["modelId"]
         
+        model_id = request.form["modelId"]
         # take a snapshots of driver.py, score.py, requirements.txt and conda_dependencies.yml
         correlation_guid = str(uuid.uuid4())
         driver_url = create_snapshot('notebooks', None, 'driver.py', 'o16n', correlation_guid)
         score_url = create_snapshot('notebooks', None, 'score.py', 'o16n', correlation_guid)
         schema_url = create_snapshot('notebooks', None, 'service_schema.json', 'o16n', correlation_guid)
         requirements_url = create_snapshot('notebooks', 'aml_config', 'requirements.txt', 'o16n', correlation_guid)
         conda_dependencies_url = create_snapshot('notebooks', 'aml_config', 'conda_dependencies.yml', 'o16n', correlation_guid)
+
+        if score_url is None or driver_url is None or schema_url is None:
+            resp = Response("One of the driver.py, score.py, service_schema.json is missing. Please run the operationalization notebook before proceeding with this step. Instructions to operationalize the model is available in the Analytics tab.", status = 400)
```

Comment:
I mean, report what is _actually_ missing. Don't just list all possible options. That doesn't help.

Reply:
I recommend throwing an exception from create_snapshot, catching that exception and then simply forwarding it.
For example:
try:
    a = create_snapshot(..., 'a.py')
    b = create_snapshot(..., 'b.py')
    c = create_snapshot(..., 'c.py')
except Exception as e:
    resp = e.message (or whatever is the way in Python to extract that message)


assuming that b.py is missing, the exception could be: "Missing b.py"